opsdroid/tests/test_logging.py

```diff
@@ -21,43 +22,41 @@ def test_set_logging_level():
 
 
 def test_configure_no_logging():
-    config = {"logging": {"path": False, "console": False}}
+    config = {"path": False, "console": False, "rich": False}
     opsdroid.configure_logging(config)
     rootlogger = logging.getLogger()
+
     assert len(rootlogger.handlers) == 1
     assert isinstance(rootlogger.handlers[0], logging.StreamHandler)
     assert rootlogger.handlers[0].level == logging.CRITICAL
 
 
 def test_configure_file_logging():
     with tempfile.TemporaryDirectory() as tmp_dir:
-        config = {
-            "logging": {"path": os.path.join(tmp_dir, "output.log"), "console": False}
-        }
+        config = {"path": os.path.join(tmp_dir, "output.log"), "console": False}
+
     opsdroid.configure_logging(config)
     rootlogger = logging.getLogger()
     assert len(rootlogger.handlers), 2
     assert isinstance(rootlogger.handlers[0], logging.StreamHandler)
-    assert rootlogger.handlers[0].level, logging.CRITICAL
-    assert isinstance(rootlogger.handlers[1], logging.handlers.RotatingFileHandler)
-    assert rootlogger.handlers[1].level == logging.INFO
+    assert isinstance(rootlogger.handlers[0], logging.handlers.RotatingFileHandler)
```

Comment:
Hmm. How can `rootlogger.handlers[0]` be both a `logging.StreamHandler` (line 41) and `logging.handlers.RotatingFileHandler` (new line 42)?

Reply:
Apparently because the RotatingFileHanlder is a StreamHandler. Either way, I changed that to check if the second item is the rich handler.